extensions/skyportal/skyportal/handlers/api/alert.py

```diff
@@ -617,19 +722,27 @@ def post(self, objectId):
             return self.error(f"failure: {_err}")
 
 
-class ZTFAlertAuxHandler(ZTFAlertHandler):
+class AlertAuxHandler(AlertHandler):
     @auth_or_token
-    def get(self, objectId: str = None):
+    def get(self, object_id: str = None):
         """
         ---
         single:
           description: Retrieve aux data for an objectId from Kowalski
+          tags:
+            - alerts
+            - kowalski
           parameters:
             - in: path
-              name: objectId
+              name: object_id
```

Comment:
Should probably decide whether we want to stick to camel or snake casing in the query params because we have both. I don't really have a preference but maybe better to stick to camel since that seems less disruptive to current users?

Reply:
I'm talking more about just what's actually put in the request, btw - the corresponding Python variables should definitely be snake case to match Python standards.

Reply:
These path parameters are positional, not explicitly named in the request URL, so not sure it matters/applies here @kmshin1397

Reply:
Ah right I missed that, you're right

Reply:
yea, I don't think it matters here. I'd keep it camel case, not to confuse those who will be reading the API specs.